### PR TITLE
This is pull request from staging to master

### PR DESCRIPTION
**Pull request by Siddhant-Damane**

### CODE REVIEW

File: commands_Siddhant.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
+1.Create and Checkout a New Branch
+
+#branches from currently checked out directory
+
+git checkout -b <branchName>
+
+2.Configure folder
```
Comment (harshal-webonise) [inline commands_Siddhant.txt:7]: whats is mean by config folder

File: commands_Siddhant.txt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+git config --global user.name "Siddhant"
+
+git config --global user.email siddhant@example.com
+
+
+3.Create a new local repository
```
Comment (harshal-webonise) [inline commands_Siddhant.txt:14]: Add proper description for each command

Comment (Siddhant-Damane) [inline commands_Siddhant.txt:14]: i.e to Configure the author name and email address to be used with commits.

**Comment by harshal-webonise** [comment]
1.Pull request title should be proper
2.Add description for pull request
3.Add more details for each commands.
4.Commit message should be proper.

**Pull request closed without merging**